gce-containers-startup/utils/run-mode.go

```diff
@@ -186,63 +189,30 @@ func createContainer(dockerClient DockerApiClient, spec api.ContainerSpecStruct)
 		runArgs = dockerstrslice.StrSlice(container.Args)
 	}
 
+	containerVolumeBindingConfigurationMap, volumePrepareError := prepareVolumesAndGetBindings(spec)
+	if volumePrepareError != nil {
+		return "", volumePrepareError
+	}
+	volumeBindingConfiguration, volumeBindingFound := containerVolumeBindingConfigurationMap[container.Name]
+	if !volumeBindingFound {
+		return "", fmt.Errorf("Volume binding configuration for container %s not found in the map, WHY??", container.Name)
```

Comment:
What about dropping "WHY??"? This could be externally visible error.

Reply:
Changed to "this should not happen", is that OK?

Reply:
Yes, thanks.